README.md

```diff
@@ -194,6 +194,54 @@ JSON Output:
 }
 ```
 
+#### Custom Search With Filters <!-- omit in toc -->
+
+If you want to enable filtering, you must add your attributes to the `FilterableAttributes` index setting.
+
+```c#
+await index.UpdateSettingsAsync(
+    new Settings
+    {
+        FilterableAttributes = new string[] { "id", "genres" },
+    }
+);
```

Comment:
There should be a specific method for this setting normally: `UpdateFilterableAttributesAsync`, also it could be great to have a variable that takes the return like `TaskInfo result = `.